.github/workflows/lib-injection.yml

```diff
@@ -94,15 +94,19 @@ jobs:
             --network test-inject \
             -p 18080:18080 \
             -e PYTHONPATH=/lib-injection \
+            -e DD_TRACE_AGENT_URL=http://testagent:8126 \
             -v $PWD/lib-injection:/lib-injection \
             ${{matrix.variant}}
           # Wait for the app to start
-          sleep 5
+          sleep 20
           docker logs ${{matrix.variant}}
       - name: Test the app
         run: |
           curl http://localhost:18080
           sleep 1  # wait for traces to be sent
-      - name: Check test agent
+      - name: Print traces
+        run: curl http://localhost:8126/test/traces
```

Comment:
Is this step necessary?

Reply:
I think it helps debugging, the step below doesn't print the traces